products/views.py

```diff
@@ -1,3 +1,23 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http       import JsonResponse
+from django.views      import View
+
+from products.models   import Product, ProductImage
```

Comment:
```suggestion
from products.models   import Product
```
ProductImage는 사용하지 않고 잇는 import 아닌가요?